site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,6 +16,27 @@ updated: 2021-11-24
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### 2022.01.05: `match_origin_as_fallback` in Canary {: #canary-match-origin-as-fallback }
+
+Content scripts can now specify the `match_origin_as_fallback` key to inject into frames that are
+related to a matching frame, including frames with `about:`, `data:`, `blob:`, and `filesystem:`
+URLs.  See the [content scripts](/docs/extensions/mv3/content_scripts/#injecting-in-related-frames)
+documentation for details.
+
+### 2021.12.30:  in Canary {: #canary-file-access }
+
+Service worker based Manifest V2 and Manifest V3 extensions can now use the Fetch API to request
```

Comment:
Optional: Since this is basically "all extensions", you could omit the "Manifest V2 and Manifest V3" and instead just say "Service-worker based extensions can now...".  Up to you, though.

Reply:
Given that V2 is deprecated, readers might not assume this applied to both V2 and V3 unless you state it explicitly. If frankly surprises me that V2 even got a new feature.

Reply:
I agree with @jpmedley's comments. I don't think it's well known that MV2 extensions can use service workers, so I wanted to make that explicit here.